modules/formulize/include/formdisplay.php

```diff
@@ -883,25 +928,75 @@ function displayForm($formframe, $entry="", $mainform="", $done_dest="", $button
 					$form->addElement($subLinkUI);
 				}
 			}
-		} 

```

Comment:
Another case where it looks like the added code includes some of the removed code?

Reply:
same as above.